sbt/src/sbt-test/source-dependencies/macro/build.sbt

```diff
@@ -0,0 +1,21 @@
+val defaultSettings = Seq(
+  libraryDependencies <+= scalaVersion("org.scala-lang" % "scala-reflect" % _ )//,
+  //incOptions := incOptions.value.withNameHashing(true)
+)
+
+lazy val root = (project in file(".")).
+  aggregate(macroProvider, macroClient).
+  settings(
+    defaultSettings
+  )
+
+lazy val macroProvider = (project in file("macro-provider")).
```

Comment:
Changing the id to be lhs is usually ok for integration tests. For real projects I would probably add `name` to make sure the artifact names remain the same.